src/providers/patreon.js

```diff
@@ -0,0 +1,27 @@
+/** @type {import(".").OAuthProvider} */
+export default function Patreon(options) {
+    return {
+        id: 'patreon',
+        name: 'Patreon',
+        type: 'oauth',
+        version: '2.0',
+        authorization: { 
+          url : 'https://www.patreon.com/oauth2/authorize?response_type=code',
+          params: { scope: 'identity identity[email] identity.memberships', grant_type: 'authorization_code' }
+        },
+        token : {
+          url : 'https://www.patreon.com/api/oauth2/token',
+        },
+        userinfo : {
+          url: 'https://www.patreon.com/api/oauth2/api/current_user'
+        },      
+        profile: (profile) => {
+          return {
+            id: profile.data.id,
+            name: profile.data.attributes.full_name,
+            email: profile.data.attributes.email
+          }
+        },
```

Comment:
```suggestion
        profile(profile) {
          return {
            id: profile.data.id,
            name: profile.data.attributes.full_name,
            email: profile.data.attributes.email,
            image: null,
          }
        },
```

So we don't receive a profile image/picture? Could you check? If so, let's set `image: null`.